test/f90_correct/lit/fc30.sh

```diff
@@ -7,3 +7,5 @@
 
 # RUN: KEEP_FILES=%keep FLAGS=%flags TEST_SRC=%s MAKE_FILE_DIR=%S/.. bash %S/runmake | tee %t 
 # RUN: cat %t | FileCheck %S/runmake
+
+# UNSUPPORTED: powerpc64le-host, ppc64le-host
```

Comment:
@bryanpkc I see powerpc64-unknown-linux-gnu, powerpc64, powerpc, powerpc64-, powerpc64le when i search for UNSUPPORTED and powerpc. Not sure what is the right string to use here.

ExecutionEngine/OrcMCJIT/remote/eh.ll:; UNSUPPORTED: powerpc64-unknown-linux-gnu
LTO/X86/parallel.ll:; UNSUPPORTED: mips, mipsel, aarch64, powerpc64
DebugInfo/debuglineinfo-path.ll:; UNSUPPORTED: powerpc
CodeGen/ppc-mm-malloc.c:// UNSUPPORTED: !powerpc64-
CodeGen/ppc-mm-malloc-le.c:// UNSUPPORTED: !powerpc64le-

Reply:
@kiranchandramohan The LLVM tests rely on feature tags of the form `$arch-registered-target`, e.g. the tags `aarch64-registered-target`, `powerpc-registered-target`, and `x86-registered-target` would be available if LLVM was built with `-DLLVM_TARGETS_TO_BUILD="AArch64;PowerPC;X86"`. These tags are only good for tests which produce IR or assembly code that do not need to be executed. In contrast, most tests under test/f90_correct/ require actual execution. For this reason, I added a new feature tag (only for the flang project) `$arch-host` in commit a97aa8cfd1fd37be2b249992bc308a633242875f, which indicates the run-time dependence of a test on the compiler's host architecture.

Reply:
The value of `$arch` in the `$arch-host` feature tag comes from `llvm-config --host-target`. I am not sure whether `llvm-config` would report `powerpc64le` or `ppc64le`, so I included both on the `UNSUPPORTED` line in the test file.